### PR TITLE
Activated GH projects and GH discussions for repository 'opendut'.

### PR DESCRIPTION
**Pull request by kKdH**

### CODE REVIEW

**Comment by netomi** [comment]
Can we get some additional approval by fellow committers / project-leads? The self-service has just been activated recently and its good if the community is aware of the changes that are happening.

**Comment by jbaumgartl** [comment]
Hi @netomi, it is approved.

For the future: @kKdH can be seen as fellow committer.

BR,
Joe (Project Lead)

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
changes are live.

In the future we want to have automatic approval for most changes with custom rules per project, where each project can declare how they want changes to be approved (e.g. require a project lead, or another committer, ...). For the time being its good to learn how each project wants to deal with that.

**Pull request merged**
